src/config.cc

```diff
@@ -282,9 +303,20 @@ void Config::updateFromPropertyTree(const boost::property_tree::ptree& pt) {
 
   // uptane.secondary_configs currently can only be set via command line.
 
+  std::string package_manager = "none";
```

Comment:
What was your thinking behind making this "none" rather than "ostree"?

Reply:
That's just the default if nothing is read out of ostree.type. Could be ostree if you'd prefer. In fact, that probably makes more sense.